server/anchor.py

```diff
@@ -63,35 +63,52 @@
 # Sets the time between transaction fetches (updates); in seconds.
 RESYNC_TIME = int(os.getenv("RESYNC_TIME", "120"))
 
-GENESIS_FILE = (
-    os.getenv("GENESIS_FILE") or "/home/indy/ledger/sandbox/pool_transactions_genesis"
-)
+# Sets the number of pool connection attempts on pool timeout.
+POOL_CONNECTION_ATTEMPTS = int(os.getenv("POOL_CONNECTION_ATTEMPTS", "5"))
+
+# Sets the delay between pool connection attempts; in seconds
+POOL_CONNECTION_DELAY = int(os.getenv("POOL_CONNECTION_DELAY", "10"))
+
+GENESIS_FILE = os.getenv("GENESIS_FILE") or "/home/indy/ledger/sandbox/pool_transactions_genesis"
 GENESIS_URL = os.getenv("GENESIS_URL")
 GENESIS_VERIFIED = False
 
-ANONYMOUS = os.getenv("ANONYMOUS")
-ANONYMOUS = bool(ANONYMOUS and ANONYMOUS != "0" and ANONYMOUS.lower() != "false")
 LEDGER_SEED = os.getenv("LEDGER_SEED")
-if not LEDGER_SEED and not ANONYMOUS:
-    LEDGER_SEED = "000000000000000000000000Trustee1"
 
-REGISTER_NEW_DIDS = os.getenv("REGISTER_NEW_DIDS", False)
-REGISTER_NEW_DIDS = bool(
-    REGISTER_NEW_DIDS
-    and REGISTER_NEW_DIDS != "0"
-    and REGISTER_NEW_DIDS.lower() != "false"
-)
+REGISTER_NEW_DIDS = env_bool("REGISTER_NEW_DIDS", False)
 
 AML_CONFIG = os.getenv("AML_CONFIG_FILE", "/home/indy/config/aml.json")
 TAA_CONFIG = os.getenv("TAA_CONFIG_FILE", "/home/indy/config/taa.json")
 
 
-def is_int(val):
-    if isinstance(val, int):
-        return True
-    if isinstance(val, str) and val.isdigit():
-        return True
-    return False
+def format_validator_info(node_data):
+    node_aliases = list(node_data.keys())
+    node_aliases.sort()
+
+    ret = []
+    for node in node_aliases:
+        try:
+            reply = json.loads(node_data[node])
+        except json.JSONDecodeError:
+            data = {"name": node, "error": node_data[node]}  # likely 'timeout'
+        else:
+            if "result" in reply:
+                data = reply["result"]["data"]
+                data["Node_info"]["Name"] = node
+            elif "reason" in reply:
+                data = {"name": node, "error": reply["reason"]}
+            else:
+                data = {"name": node, "error": "unknown error"}
+        ret.append(data)
+    return ret
```

Comment:
```suggestion
def format_validator_info(node_data):
    node_aliases = list(node_data.keys())
    node_aliases.sort()

    ret = []
    for node in node_aliases:
        try:
            reply = json.loads(node_data[node])
        except json.JSONDecodeError:
            data = {"Node_info": {"Name": node}, "error": node_data[node]}  # likely 'timeout'
        else:
            if "result" in reply:
                data = reply["result"]["data"]
            elif "reason" in reply:
                data = {"Node_info": {"Name": node}, "error": reply["reason"]}
            else:
                data = {"Node_info": {"Name": node}, "error": "unknown error"}
        ret.append(data)
    
    return ret
```

The name of the node needs to be placed into `Node_info.Name` for `/status/text` to work when there is an error result.

Updates are also required to `app.methods.formatValidatorStatus` in `server/static/include/js/index.js` to handle the error results.

Adding:
```
          if (node.error)
            continue
```
to the `for` loop will retain the existing display functionality when nodes encounter an error condition.  In the following example Node 2 was stopped:

![image](https://user-images.githubusercontent.com/10504350/215218630-7e120b83-bc5d-4d98-be89-8d45b8e80069.png)

Reply:
Good catch. Done.